Makefile

```
@@ -56,6 +56,11 @@ ifeq ($(BASE_DIR),connectathon)
 	cd connectathon && git checkout 8fbaf58758a5d0719dd4b526f5018a5b33d8272e
 endif
 
+VALUESET_FILES = $(shell find $(BASE_DIR)/fhir4/bundles -type f -name "valuesets*bundle.json")
+.copy-valuesets:
```

NIT: Should probably require connectathon and synthea

```suggestion
.copy-valuesets: connectathon synthea
```

---

Agreed